src/test/java/com/ays/admin_user/service/impl/AdminUserRegisterServiceImplTest.java

```diff
@@ -96,12 +87,10 @@ void givenValidAdminUserRegisterRequest_thenAdminUserRegistered() {
                 .thenReturn(mockCompletedVerificationEntity);
 
         // Then
-        adminUserRegisterService.register(mockAdminUserRegisterRequest);
+        adminUserRegisterService.completeRegistration(applicationId, mockAdminUserRegisterApplicationCompleteRequest);
 
         Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Comment:
Aşağıdaki gibi suggestion kısmını da ekleyebilir miyiz?
```suggestion
        // Verify
        Mockito.verify(adminUserRegisterApplicationRepository, Mockito.times(1))
```

Reply:
Ekledim.

Reply:
✔ Bu kısmı **Resolve Conversation** edebiliriz.